MANUAL.md

```diff
@@ -0,0 +1,202 @@
+            Type: source
+            Repo: CRAN_Micro
+    ```
+  - Repository-specific customizations
+    - Installation type for packages pulled from a repo
+    - Whether or not to install Suggested dependencies for packages pulled from a repo
+    - Yaml syntax:
+    ```
+    Customizations:
+      Repos:
+        - <repo_name as listed in this config file>:
+            Type: <source/binary>
+        - <repo_name as listed in this config file>:
+            Suggests: <true/false>
+        - CRAN_Micro:
+            Type: source
+            Suggests: true
+    ```
+* Libpaths [[ I don't know what this does]]
+* Rpath [[I don't know what this does in terms of pkgr]]
+* Cache directory
+  - Directory to use for storing cached files files
+  - Folder must exist
+  - Yaml syntax:
+  ```
+  Cache: <path_to_desired_folder
+  ```
+* Logging Output
+  - Settings for logging output.
+  - Yaml syntax:
+  ```
+  Logging:
+    all: <path_to_file>
+    install: <path_to_file>
+    overwrite: <true/false>
+  ```
+  - These configuration options are slated for rework, but for now, they behave as described below:
+    * Setting: `all: x.log`
+      * `pkgr install` logs to x.log when `install=y.log` is not set.
+      * `pkgr clean --all` logs to x.log
+      * `pkgr inspect` logs to x.log
+      * `pkgr clean cache` and `pkgr clean pkgdbs` do NOT log to x.log
+      * `pkgr plan` does NOT log to x.log
+    * Setting: `install: y.log`
+      * `pkgr install` logs to y.log (and only to y.log)
+    * Setting: overwrite: `true`
+      * Logs are overwritten every time a new pkgr command is run.
+      * If false, logs are appended to every time a new pkgr command is run.
+
+* Lockfile settings
+  - Lockfile to look for, specified by its type. Pkgr supports Packrat and Renv lockfiles.
+  - Yaml syntax:
+  ```
+  Lockfile:
+    Type: [packrat/renv]
+  ```
+
+### The following items are configurable statically in the yml file or dynamically via the command-line.
+* **Note:** Command-line flags will always overwrite the configuration in the yaml file.
+
+* Library (directory) to install packages to
+  - Yaml syntax
+  ```
+  Library: <path_to_directory>
+  ```
+* Update settings (should pkgr attempt to update outdated packages?)
+  - Yaml syntax
+  ```
+  Update: <true/false>
+  ```
+* Logging level
+  - Yaml syntax
+  ```
+  Loglevel: <info/debug/trace>
+  ```
+* Installation of “Suggested” packages config
+  - Yaml syntax
+  ```
+  Suggested: <true/false>
+  ```
+* Number of threads to use
+  - Yaml syntax
+  ```
+  Threads: <number_of_threads>
+  ```
+* Strict mode enabled/disabled
+  - Yaml syntax
+  ```
+  Strict: <true/false>
+  ```
+
+## Default Settings
+Below are the default settings for non-required pkgr configurations.
+  * Debug mode: disabled
+  * Preview mode: disabled
+  * Strict mode: disabled
+  * Loglevel: “info”
+  * Rpath: “R”
+  * Threads: Number of processors available on system
+  * Cache: System temp directories
```

Comment:
User cache directory (not system) - under the hood this uses ~ os.UserCacheDir() which is I believe like ~/.cache/... on unix systems

Reply:
> this is looking really good/helpful. The one thing is I think this needs to either be in the wiki _or_ the readme. I doubt people will find the manual (I'm not aware of other repos that have a manual.md), so if we're going to go off the readme, we might as well use a "real" system - aka the wiki, that can be maintained/extended as large as we need it to be.

I agree that the Wiki is the place to put this ultimately, I'll transfer the information over there once we've gotten everything straight with the testing docs, as those are currently cluttering it up.